package.json

```diff
@@ -14,6 +14,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.13.0",
     "@fortawesome/react-fontawesome": "^0.1.9",
     "highlight.js": "^9.18.1",
+    "nan": "^2.14.1",
```

Comment:
I see that you updated image file names. But is it necessary to add nan package?

Reply:
When I was running `npm install` it showed an error saying **_Cannot find module 'nan'_**, so I installed it to make it run, I'm not sure if it's missing in the package.json but I can remove it if necessary

Reply:
This is the error message

![Screen Shot 2020-10-05 at 7 34 08 PM](https://user-images.githubusercontent.com/2978730/95158038-445afa80-0760-11eb-9119-f0c7ce95ee73.png)

Reply:
What OS are you using?

Reply:
macOS Catalina 10.15.6

Reply:
I found better solution! 🥇  You can install nan globally. And it won't affect this project and you won't have problem with other project.

`
npm i -g nan
`

Have a look at: [Error: Cannot find module 'nan'](https://stackoverflow.com/questions/34464673/error-cannot-find-module-nan)

Try it and have a look 😃

Reply:
You are right, it worked 🙌🏻

I removed the dependency from _package.json_

Thanks 😃